packages/ses/scripts/hermesTransforms.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Please rename the file `hermes-transforms.js` to be consistent with the prevailing kebab aesthetic.